packages/grafana-ui/src/components/Monaco/types.ts

```diff
@@ -1,10 +1,12 @@
 // We use `import type` to guarentee it'll be erased from the JS and it doesnt accidently bundle monaco
-import type * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
+import type * as monacoType from 'monaco-editor/esm/vs/editor/editor.api';
 
 export type CodeEditorChangeHandler = (value: string) => void;
 export type CodeEditorSuggestionProvider = () => CodeEditorSuggestionItem[];
 
-export type MonacoEditor = monaco.editor.IStandaloneCodeEditor;
+export type Monaco = typeof monacoType;
+export type MonacoEditor = monacoType.editor.IStandaloneCodeEditor;
+export type MonacoOptions = monacoType.editor.IStandaloneEditorConstructionOptions;
```

Comment:
I'm not super in love with the typescript work I've done here, so I would love some thoughts on it. Preferably all monaco types would be exported from this file, and elsewhere in our code it would import it from here, but we use types like `monacoType.languages.CompletionItemKind` which is hard to export (you can't export a namespace?), so I would love thoughts on all of this.